### PR TITLE
Added travis-ci and fixed coding standards issues

### PR DESCRIPTION
**Pull request by christianselig**
Added travis-ci to the module and fixed coding standards issues.

### CODE REVIEW

File: islandora_marcxml [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 1670b22a4fc91009ada99a897b68781b29b65e35
```
Comment (jonathangreen) [inline islandora_marcxml:1]: Not sure what this is all about?

**Comment by jonathangreen** [comment]
Everything but that weird file looks good for this one.

**Comment by jonathangreen** [comment]
This looks like it might have a merge conflict upstream, that should be resolved before I can merge as well. Can you merge in the upstream changes @christianselig.

**Comment by christianselig** [comment]
Did it! I think. Git's a lot of fun, yay for learning.

**Comment by jonathangreen** [comment]
Looks like travis is failing again.

**Comment by christianselig** [comment]
On it.

**Comment by christianselig** [comment]
Should be fully fixed now.

**Comment by jonathangreen** [comment]
Can you remove that islandora marcxml file from the pull request? thats the last thing before this one gets merged.

**Pull request closed without merging**